packages/flutter/lib/src/widgets/route_notification_messages.dart

```diff
@@ -14,6 +15,15 @@ class RouteNotificationMessages {
   // ignore: unused_element
   RouteNotificationMessages._();
 
+  /// When the engine is Web notify the platform for a route information change.
+  static void maybeNotifyRouteInformationChange(RouteInformation routeInformation, RouteInformation previousRouteInformation) {
```

Comment:
We should always send this information to the engine, it's up to the embedder to ignore or process it.